pantry-finder-api/app/serializers/event_date_serializer.rb

```diff
@@ -4,8 +4,8 @@
 class EventDateSerializer < ActiveModel::Serializer
   attributes :id
 
-  attribute :service do
-    object.event.service_description
+  attribute :event_id do
+    object.event.event_id
```

Comment:
Do we need this anymore? Since this is now nested under events, this data is unused.

Reply:
Have a similar view as with loc_id.  You can go too far.  But It can also be a troubleshooting tool as well as a kind of linked data for clients.